lib/src/io.dart

```diff
@@ -842,10 +842,28 @@ Future<HttpServer> bindServer(String host, int port) async {
   return server;
 }
 
+/// Find a tar. Prefering system installed tar.
+///
+/// On linux tar should always be /bin/tar [See FHS 2.3][1]
+/// On MacOS it seems to always be /usr/bin/tar.
+///
+/// [1]: https://refspecs.linuxfoundation.org/FHS_2.3/fhs-2.3.pdf
+String _findTarPath() {
+  for (final file in ['/bin/tar', '/usr/bin/tar']) {
+    if (File(file).existsSync()) {
```

Comment:
nit: use `fileExists(file)` defined in this file and use throughout pub.

Reply:
done